tests/test_ensemble_tools.py

```diff
@@ -69,6 +70,30 @@ def _check_parameter_in_conditions(
     return (n_equals, n_different)
 
 
+def test_check_parameter_in_conditions(tmp_path: Path) -> None:
+    """Test that the private xml parser method of this class."""
```

Comment:
This docstring doesn't read quite right - possibly an additional `that`?

Reply:
Yep, I think I changed my mind on what it was gonna say half way through writing it - woops! Fixed now